packages/cli/src/controllers/owner.controller.ts

```diff
@@ -148,4 +148,12 @@ export class OwnerController {
 
 		return sanitizeUser(owner);
 	}
+
+	@Post('/dismiss-v1')
+	async dismissBanner() {
+		await this.settingsRepository.saveSetting('ui.banners.v1.dismissed', JSON.stringify(true));
+		this.config.set('ui.banners.v1.dismissed', true);
```

Comment:
perhaps this should also be in `settingsRepository.saveSetting` (if `loadOnStartup` is `true`)